projects/charger/src/begin_sequence.c

```diff
@@ -0,0 +1,71 @@
+#include "begin_sequence.h"
+
+#include <stdbool.h>
+#include <stdint.h>
+
+#include "can.h"
+#include "can_transmit.h"
+#include "charger_controller.h"
+#include "charger_events.h"
+#include "control_pilot.h"
+#include "event_queue.h"
+#include "exported_enums.h"
+#include "gpio.h"
+#include "status.h"
```

Comment:
Just a comment: I was thinking that we should have a standardized convention for #include order, probably like what you have here: this module's header (if applicable), then stdlib, then our libs, each section sorted alphabetically and separated by a blank line.

Reply:
true, can you add this to the styleguide?